src/pricing-fetcher.test.ts

```
@@ -252,4 +252,47 @@ describe('pricing-fetcher', () => {
 			expect(cost).toBe(0);
 		});
 	});
+
+	describe('offline mode', () => {
```

ESLint

[error] Unsafe call of a(n) `error` type typed value.

(ts/no-unsafe-call)